server/api/helpers/harvestGames/get-hg-rankings

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
i dont think you need this entire file? you are not actually using the helper function at all anywhere. also the file is not even a js file

Reply:
didn't realize haha, will remove it